lib/archethic/transaction_chain/transaction/proof_of_validation.ex

```diff
@@ -0,0 +1,178 @@
+defmodule Archethic.TransactionChain.Transaction.ProofOfValidation do
+  @moduledoc """
+  Handle the Proof Of Validation signatures containing
+  - The aggregated signatures
+  - Nodes bitmask (bitmask of nodes used for the signatures)
+
+  Proof of Validation aggregate all valid Cross Validation Stamp signatures
+  It require a threshold of cross stamp without error based on the expected number
```

Comment:
Suggestion: I think you need to alter the description as it's not based on threshold anymore.